sycl/test/regression/ctz_namespace.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clangxx -fsycl -fsycl-device-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning %s
+
+// expected-no-diagnostics
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -S -emit-llvm -fsycl-device-only %s -o - -Xclang -disable-llvm-passes | FileCheck %s
```
The following change is need to address comment below